### ISSUE TITLE
** Feature Request ** UIkit3 Range Slider Component - Design Proposal

### ISSUE THREAD

**Issue by zzseba78** [uikit/uikit#2625]
UIkit3 Range Slider Component
Thanks for your work.

![rangeslider](https://cloud.githubusercontent.com/assets/5558563/25363070/8033deb4-292e-11e7-966c-9349f6cb7ad7.png)

**Comment by carlos-duran** [comment]
I vote for this!! I'ts very usefull :+1:

**Comment by zzseba78** [comment]
Seems that it´s coming soon :)

**Comment by mvanvu** [comment]
Nice!

**Comment by ruchern** [comment]
I believe it's now implemented. Great work!

**Comment by Riddler84** [comment]
No, it's not. We've only get a styling for the html5 range input, but no javascript implementation. So theres no tooltips, scale labels or double slider yet. :/

**Comment by zzseba78** [comment]
Nope... :(

**Comment by danijelh** [comment]
I came here looking for exactly this. I am still in the process of choosing which framework to go with and this is one of the features I need. I like UiKit so far and would be really gr8 if range slider would be implemented!

**Comment by zzseba78** [comment]
@danijelh Besides this feature it´s implemented or not, go for this frameworks, it´s amazing.

**Comment by arisaves** [comment]
@danijelh I believe there actually is a range slider! I am not sure if this one suits your needs, but you can check it out under the **_Legend_** section at https://getuikit.com/docs/form -- I believe you'd add the `uk-range` class to an input `range` element. I'm about to try it myself, I'll let you know how it goes.

**Comment by zzseba78** [comment]
@arisaves Yes, it is, but it doesn´t have the use of requested feature. It only let´s you pick one value, not a range.

**Comment by arisaves** [comment]
thanks for clarifying @zzseba78

**Comment by yaneony** [comment]
Here is an example of uk-range with tooltips added:
https://uikit.plus/snippets/range-slider-tooltip-67dc1c5b7f08dfeef6ab0f45
